internal/databroker/config_source.go

```diff
@@ -272,15 +272,8 @@ func (src *ConfigSource) runUpdater(cfg *config.Config) {
 	}, databroker.WithTypeURL(grpcutil.GetTypeURL(new(configpb.Config))),
 		databroker.WithFastForward())
 	go func() {
-		var databrokerURLs []string
-		urls, _ := cfg.Options.GetDataBrokerURLs()
-		for _, url := range urls {
-			databrokerURLs = append(databrokerURLs, url.String())
-		}
```

Comment:
I'm not sure I understand this part. Is this logging no longer correct if the gRPC listener is disabled?

Reply:
It will show:

```json
{
  "level":"debug",
  "outbound_port":"51313",
  "databroker_urls":["http://127.0.0.1:5443"],
  "time":"2024-04-10T11:31:30-06:00",
  "message":"config: starting databroker config source syncer"
}
```

But that's not actually correct. I don't think this log was particularly useful so I thought it made more sense to just remove it rather than fixing it.